ui/src/app/utils/user-apps-utils.tsx

```diff
@@ -68,37 +73,33 @@ export const createUserApp = (namespace, config: CreateAppRequest) =>
       maybeStartPollingForUserApps(namespace);
     });
 
-export const deleteUserApp = (namespace, appName, deleteDiskWithUserApp) => {
-  return appsApi()
+export const deleteUserApp = (namespace, appName, deleteDiskWithUserApp) =>
+  appsApi()
     .deleteApp(namespace, appName, deleteDiskWithUserApp)
     .then(() => maybeStartPollingForUserApps(namespace));
-};
 
-export const pauseUserApp = (googleProject, appName, namespace) => {
+export const pauseUserApp = (googleProject, appName, namespace) =>
```

Comment:
not sure if these return values are useful, but let's give callers the option of using them

Reply:
I think that's a good idea but I'm running into issues that I don't think I can solve quickly - so I'm going to hold off on this for another time.